.github/workflows/run_tests.yml

```diff
@@ -29,10 +29,11 @@ jobs:
         cc: ['gcc', 'clang']
         buildtype: ['debug', 'release']
         include:
+          # TODO: re-enable coverage tests once we figure out why they're failing
```

Comment:
Could you add a link here to #1152 for reference?

Reply:
Added